metrixpp/ext/std/tools/collect.py

```diff
@@ -46,14 +46,23 @@ def configure(self, options):
         self.is_proctime_enabled = options.__dict__['std.general.proctime']
         self.is_procerrors_enabled = options.__dict__['std.general.procerrors']
         self.is_size_enabled = options.__dict__['std.general.size']
-        try:
-            self.add_include_rule(re.compile(options.__dict__['include_files']))
-        except Exception as e:
-            self.optparser.error("option --include-files: " + str(e))
-        try:
-            self.add_exclude_rule(re.compile(options.__dict__['exclude_files']))
-        except Exception as e:
-            self.optparser.error("option --exclude-files: " + str(e))
+        if options.__dict__['include_files']:
+            print(options.__dict__['include_files'])
+            try:
+                for include_rule in options.__dict__['include_files']:

```

Comment:
how does it know if the option is given once or many times? Will it be always an array?

Reply:
'include_files' is None if no rule is given and contains a list of rules if one ore more rules are passed.

So if there is no rule passed by the user include_files are None and the else path is taken which applies the default rule.